src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/SpecialFileProviders/AbstractFindByNameSpecialFileProvider.cs

```diff
@@ -29,8 +28,8 @@ internal abstract class AbstractFindByNameSpecialFileProvider : ISpecialFileProv
 
         protected AbstractFindByNameSpecialFileProvider(
             IPhysicalProjectTree projectTree,
-            [Import(ExportContractNames.ProjectItemProviders.SourceFiles)] IProjectItemProvider sourceItemsProvider,
-            [Import(AllowDefault = true)] Lazy<ICreateFileFromTemplateService>? templateFileCreationService,
+            IProjectItemProvider sourceItemsProvider,
+            Lazy<ICreateFileFromTemplateService>? templateFileCreationService,
```

Comment:
This was only nullable due to `AllowDefault = true`:

```suggestion
            Lazy<ICreateFileFromTemplateService> templateFileCreationService,
```